### PR TITLE
Code style changes

### PR DESCRIPTION
**Pull request by sharwell**
📝 Extracted from #30733

### CODE REVIEW

**Comment by mavasani** [comment]
Should this go into master instead of dev16.0.x?

**Comment by CyrusNajmabadi** [comment]
this seems very stylistic.  is there as reason it's going into dev16?

File: src/Features/Core/Portable/CodeLens/ReferenceLocationDescriptor.cs [hunk 1]
```diff
@@ -9,14 +9,20 @@ namespace Microsoft.CodeAnalysis.CodeLens
...
-        public int LineNumber { get; }
-
-        public int ColumnNumber { get; }
-
         public Guid ProjectGuid { get; }
 
         public Guid DocumentGuid { get; }
 
+        /// <summary>
+        /// Reference's Line
```
Comment (sharwell) [inline src/Features/Core/Portable/CodeLens/ReferenceLocationDescriptor.cs]: > Reference's line based on the document content

File: src/Features/Core/Portable/CodeLens/ReferenceLocationDescriptor.cs [hunk 2]
```diff
@@ -9,14 +9,20 @@ namespace Microsoft.CodeAnalysis.CodeLens
...
+        /// </summary>
+        public int LineNumber { get; }
+
+        /// <summary>
+        /// Reference's Column
```
Comment (sharwell) [inline src/Features/Core/Portable/CodeLens/ReferenceLocationDescriptor.cs]: > Reference's character based on the document content

File: src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs [hunk 3]
```diff
@@ -1145,11 +1145,11 @@ protected override void Dispose(bool finalize)
             base.Dispose(finalize);
         }
 
-        public void EnsureEditableDocuments(IEnumerable<DocumentId> documents)
+        public void EnsureEditableDocuments(IEnumerable<DocumentId> documentIds)
```
Comment (sharwell) [inline src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs]: > documents

**Comment by sharwell** [comment]
This is now updated through commit bf32f41ab80ce173521a81eb7d35ecfba99e62c8

**Comment by sharwell** [comment]
> this seems very stylistic. is there as reason it's going into dev16?

This was extracted from #30733. It's being merged first to reduce the overall size of the diff in that pull request.

**Comment by CyrusNajmabadi** [comment]
> This was extracted from #30733. It's being merged first to reduce the overall size of the diff in that pull request.

The same concern applies.  Why does `#30733` need this?

**Comment by sharwell** [comment]
@CyrusNajmabadi Pull request #30733 is not in a place where these changes can be easily eliminated from it. Merging that pull request will automatically include these changes, so my goal was to call them out as opposed to selectively reverting them.

**Comment by CyrusNajmabadi** [comment]
> @CyrusNajmabadi Pull request #30733 is not in a place where these changes can be easily eliminated from it. Merging that pull request will automatically include these changes, so my goal was to call them out as opposed to selectively reverting them.

I'm not going to push more on this topic.  However, IME, this isn't actually that hard.

You just have this PR (pushed into a feature branch), you then merge it into #30733.  You then revert this PR from the feature branch and merge that into #30733.  That PR then doesn't have this change at all.  That PR then goes into Dev16.  You then cherry pick this change into master.  It's a couple of additional steps, but it allows pulling things out of a PR and moving it into another branch withotu too much difficulty.  

That said, as per the above, this isn't a huge deal as it really is just style.  If this involved unnecessary functional changes, i would think it was more important to not do.

**Pull request merged**